source/PlayState.hx

```diff
@@ -3698,11 +3698,11 @@ class PlayState extends MusicBeatState
 								addCharacterToList(value2, charType);
 							}
 
-							var wasGf:Bool = dad.curCharacter.startsWith(gf.curCharacter);
+							var wasGf:Bool = dad.curCharacter == gf.curCharacter;
 							var lastAlpha:Float = dad.alpha;
 							dad.alpha = 0.00001;
 							dad = dadMap.get(value2);
-							if(!dad.curCharacter.startsWith(gf.curCharacter)) {
+							if(!dad.curCharacter == gf.curCharacter) {
```

Comment:
change this to `if (dad.curCharacter != gf.curCharacter)`

Reply:
Ok I'll get to it soon.

Reply:
At school rn